### PR TITLE
Minutes 2024 07 03

### PR DESCRIPTION
**Pull request by timlegge**

### CODE REVIEW

**Comment by timlegge** [comment]
@sjn I thought I had pushed this already

File: docs/minutes/cpansec-minutes-2024-07-03.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- Something with better audio quality
+- That doesn't require invites/sign-in
+
+This meeting was done in Element/[Matrix], which uses Jitsi. Went well.
+
+### CPANminus
+1. We have sent PRs to master and development
+2. An independant PR has been sent into the official perl-docker image.
+3. Trying to push for use of the new patched version
+4. What are the next steps
```
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-07-03.md:41]: point 4 doesn't say anything useful. remove?

File: docs/minutes/cpansec-minutes-2024-07-03.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+3. PRF is a foundation perl noc hosts code in theory neither can join the organization - need to do both.
+
+### CycloneDX 1.7 Sustainability fields
+1. @sjn - talked to someone who is working in the sustainability of opensource projects
+
+### CPAN Meta Requirements and PURLs
+1. @sjn - no update
+
+### POSIX::2008 vulnerabilities
+1. Need to look at register CVE for that
```
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-07-03.md:75]: Who needs to look at registering a CVE?

File: docs/minutes/cpansec-minutes-2024-07-03.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+1. [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria - September 16-18 + 19-20 - Lots of people from OpenSSF + SBOM + Supply chain security communities
+1. [London Perl Workshop](https://act.yapc.eu/lpw2024/) - Saturday 26th October 2024 - possible talk opportunities
+
+## Other topics
+1. @Tux and @Tim worked on Crypt::OpenSSL::PKCS12
+2. @stigsto how should a secure random module get moved to core?  @garu explainde the process for getting things moved into core - requests for comment, etc.
+
+## Next meeting
+1. July 18, 2024 at 17:00 UTC
+https://www.timeanddate.com/worldclock/converter.html?iso=20240718T170000&p1=1440&p2=1129&p3=136&p4=195&p5=16&p6=187&p7=233&p8=37&p9=250&p10=234&p11=256&p12=248
```
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-07-03.md:93]: Remove links like these from the minutes

File: docs/minutes/cpansec-minutes-2024-07-17.md
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-07-17.md:1]: Date in filename is wrong

File: docs/minutes/cpansec-minutes-2024-07-17.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+No updates
+
+### CNA process is initiated with Mitre
+
+- @timlegge has written up an initial CNA disclosure but it needs more work - hoppefully this week
+
+### TLS/HTTPS in core
+1. Options documented
+1. Currently only opensssl
```
Comment (sjn) [inline docs/minutes/cpansec-minutes-2024-07-17.md:53]: typo: opensssl

File: docs/minutes/cpansec-minutes-2024-07-03.md (hunk 3 above)
Comment (timlegge) [reply docs/minutes/cpansec-minutes-2024-07-03.md:93]: We can but I am not sure that it would not be useful if we ever get timely at posting these minutes

**Pull request merged**